app/facades/item_facade.rb

```diff
@@ -0,0 +1,17 @@
+class ItemFacade
+
+  def self.items 
+    json = ItemService.items[:data]
+  
+    @items = json.map do |item|
+      Item.new(id: item[:id], name: item[:attributes][:name], description: item[:attributes][:description], unit_price: item[:attributes][:unit_price], merchant_id: item[:attributes][:merchant_id])
```

Comment:
I think this could just be `Item.new(item)` with the correct PORO setup, but if not, consider splitting arguments onto multiple lines for readability:
```
Item.new(
  id: item[:id],
  name: item[:attributes][:name],
  description: item[:attributes][:description],
  ...
```